File: 백준/Bronze/2566. 최댓값/최댓값.c
#include <stdio.h>

int main()
{
	int a[9][9];
	int i, j;
	int max = 0; 
	int c = 0, b = 0;

	// 값 넣기 
	for (i = 0; i < 9; i++)
	{
		for (j = 0; j < 9; j++)
		{
			scanf("%d", &a[i][j]);
		}
	}
	
	// 최댓값 찾기
	for (i = 0; i < 9; i++)
	{
		for (j = 0; j < 9; j++)
		{
			if (a[i][j] > max)
			{
				max = a[i][j];
				b = i;
				c = j;
			}
		}
	}

	printf("%d\n", max);
	printf("%d %d\n", b + 1, c + 1);
	return 0;
}

File: 백준/Bronze/2566. 최댓값/최댓값.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        int max = Integer.MIN_VALUE;
        int r = 0;
        int c = 0;
        Scanner sc = new Scanner(System.in);

        int [][] arr = new int[9][9];
        for(int i = 0; i < 9; i++) {
            for(int j = 0; j < 9; j++) {
                arr[i][j] = sc.nextInt();
            }
        }

        for(int i = 0; i < 9; i++) {
            for(int j = 0; j < 9; j++) {
                if(arr[i][j] > max) {
                    max = arr[i][j];
                    r = i + 1;
                    c = j + 1;
                }
            }
        }
        System.out.println(max);
        System.out.print(r);
        System.out.print(" " + c);
    }
}
